### DISCUSSION TITLE
Watch Replay

### DISCUSSION THREAD

**Discussion by SmooothSoundz** [efroemling/ballistica#267] [Ideas]
Any chance of adding rewind, play, pause, and fast forward buttons to the Watch Replay section? Maybe even a way of saving the replay in .mp4 format...

-Smoooth

**Comment by Gengar-lab** [comment]
This has been discussed in #132.

**Comment by MatiasMGz** [comment]
This should be closed. Please close this!

**Comment by Loup-Garou911XD** [comment]
> Any chance of adding rewind, play, pause, and fast forward buttons to the Watch Replay section? Maybe even a way of saving the replay in .mp4 format...

this feature was added
https://github.com/efroemling/ballistica/pull/658
